tests/components/application-connector/resources/charts/gateway-test/charts/test/templates/applications/credentials/oauth-negative-incorrect-id.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
   namespace: kyma-integration
 type: Opaque
 data:
-  clientId: {{ "bad id" | b64enc }}
+  clientId: {{ "clientID" | b64enc }}
```

Comment:
I believe this one should not be changed :d It's the `oauth-negative-incorrect-id` case